index.html

```diff
@@ -158,6 +158,28 @@ <h1 class="about-head">About me</h1>
       </ul>
     </div>
   </section>
+
+  <section class="contact-section">
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form action="https://formspree.io/f/mknagrvq" method="POST" class="contact-form">
+      <input type="text" name="name" maxlength="30" placeholder="Full name" class="form-input" required />
+      <input type="email" name="email" placeholder="Email address" class="form-input"required />
+      <textarea maxlength="500" name="description" placeholder="Enter text here" class="form-input form-textarea" rows="5"required ></textarea>
+      <button type="submit" class="submit-btn">Get in touch</button>
+    </form>
+  </section>
+
+  <footer>
+    <ul class="footer-icons">
+      <li><a href="https://github.com/Yosefgeda"><img src="icons/GitHub.png" alt="github logo"></a></li>
+      <li><a href="www.linkedin.com/in/yosefgeda"><img src="icons/Linkedin.png" alt="linkedin logo"></a></li>
+      <li><img src="icons/Angelist.png" alt="angel loist"></li>
+      <li><a href="twitter.com/yosefgeda"><img src="icons/Twitter.png" alt="twitter logo"></a></li>
+      <li><img src="icons/Medium.png" alt="BigM"></li>
+    </ul>
+    <span class="footer-line"><hr></span>
+  </footer>
```

Comment:
- [ ] You should try to stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). You got all this well except the spacing, Your contact form lack proper margins on the left and right sides. There should be no scroll bar on the horizontal side of the browser.
- [ ] Every other thing you did well. Well done 👏 

**_Figma Template_**
![image](https://user-images.githubusercontent.com/84156714/212929489-f6759b2f-f4bd-4907-af59-a13e60d7efd4.png)

**_Your work_**
![image](https://user-images.githubusercontent.com/84156714/212930931-2dae524b-e776-496c-95f4-60d8441b2b25.png)